internal/storage/network_server_test.go

```diff
@@ -29,13 +29,15 @@ func TestNetworkServer(t *testing.T) {
 		}{
 			{
 				NetworkServer: NetworkServer{
+					Name:                     "test-ns",
 					GatewayDiscoveryEnabled:  false,
-					GatewayDiscoveryInterval: 0,
+					GatewayDiscoveryInterval: 5,
```

Comment:
Is this change relevant for this PR? If not would you mind reverting it?

Reply:
If `GatewayDiscoveryEnabled: true` then required otherwise you can revert it.

Thanks